project/src/mkdocstrings_handlers/{{python_package_import_name}}/handler.py.jinja

```diff
@@ -46,9 +46,33 @@ class {{ language|title }}Handler(BaseHandler):
     """  # noqa: E501
 
     def collect(self, identifier: str, config: dict) -> CollectorItem:  # noqa: D102
+        """Collect data given an identifier and selection configuration.
+
+        In the implementation, you typically call a subprocess that returns JSON, and load that JSON again into
+        a Python dictionary for example, though the implementation is completely free.
+
+        Arguments:
+            identifier: An identifier that was found in a markdown document for which to collect data. For example,
+                in Python, it would be 'mkdocstrings.handlers' to collect documentation about the handlers module.
+                It can be anything that you can feed to the tool of your choice.
+            config: Configuration options for the tool you use to collect data. Typically called "selection" because
+                these options modify how the objects or documentation are "selected" in the source code.
```

Comment:
mkdocstrings' own docstrings might be outdated. Now we pass every option to both `collect` and render, so there's no distinction of "selection" and "rendering" anymore. Could you update the docstring accordingly?